Userland/Libraries/LibWeb/CSS/Transformation.cpp

```diff
@@ -38,6 +43,13 @@ Gfx::FloatMatrix4x4 Transformation::to_matrix(Painting::PaintableBox const& pain
     auto height = CSS::Length::make_px(reference_box.height());
 
     switch (m_function) {
+    case CSS::TransformFunction::Perspective:
+        if (count == 1)
+            return Gfx::FloatMatrix4x4(1, 0, 0, 0,
+                0, 1, 0, 0,
+                0, 0, 1, 0,
+                0, 0, -1 / value(0), 1);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/perspective claims that `"none"` and values <= 0 are special cases.

Reply:
You are right I missed that one